language/sentiment/README.txt

```diff
@@ -0,0 +1,27 @@
+# Introduction
+
+This sample contains the code referenced in the 
+[Sentiment Analysis Tutorial](http://cloud.google.com/natural-language/docs/sentiment-tutorial)
+within the Natural Language API Documentation. A full walkthrough of this sample
+is located within the documentation.
+
+This sample is a simple illustration of how to construct a sentiment analysis
+request and process a response using the API.
+
+## Download the Code
+
+```
+git clone sso://python-dev-samples/language/sentiment/
+```
+
+### Usage
```

Comment:
<http://go/python-samples-guide#canonical-sample> has apparently been factored away, but you can check out [python-docs-samples/storage](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/storage) for its README